test/fast/Aliases/lts/"nvm alias" should ensure LTS alias dir exists

```diff
@@ -7,10 +7,23 @@ set -ex
 
 LTS_ALIAS_PATH="$(nvm_alias_path)/lts"
 
-die () { echo "$@" ; exit 1; }
+if [ -d "${LTS_ALIAS_PATH}" ]; then
```

Comment:
actually - i'm confused. There's already a `setup_dir` in the `lts` dir that backs it up just fine. If another test happens to recreate it in the meantime, it's totally fine if it gets used, or overwritten.

If you mean that, for this test to work, there must be no LTS dir, then sure, just directly delete it.